services/brig/src/Brig/User/Search/Index.hs

```diff
@@ -586,7 +588,38 @@ indexMapping =
                   mpIndex = True,
                   mpAnalyzer = Nothing,
                   mpFields = mempty
-                }
+                },
+            "scim_external_id"
+              .= MappingProperty
+                { mpType = MPKeyword,
+                  mpStore = False,
+                  mpIndex = False,
```

Comment:
On second thought, can you double-check if this is something we need for implementing the current designs?  Would be annoying if we had to make another PR, another migration...  (sorry for the back & forth, but it's better not to index things we don't need indices for, ES takes a lot of resources).

Reply:
Not sure, I understand the concern. The field is explicitly not indexed.